application/linux/.gitignore

```diff
@@ -0,0 +1 @@
+flutter/ephemeral
```

Comment:
Itt is igaz, hogy projektenként 1 gitignore file legyen, ahogy korábbi kommentben szerepel.